synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -195,49 +182,40 @@ ffmpeg_trgt::init(ProgressCallback *cb=NULL)
 	}
 
 	String ffmpeg_binary_path;
-	std::list< String > binary_choices;
-	binary_choices.push_back("ffmpeg");
-	binary_choices.push_back("avconv");
-	std::list< String >::iterator iter;
-	for(iter=binary_choices.begin();iter!=binary_choices.end();iter++)
-	{
-		String binary_path;
 #if defined(WIN32_PIPE_TO_PROCESSES)
-		binary_path = synfig::get_binary_path("");
-		if (binary_path != "")
-			binary_path = etl::dirname(binary_path)+ETL_DIRECTORY_SEPARATOR;
-		binary_path += *iter+".exe";
-		if( access( binary_path.c_str(), F_OK ) != -1 ) {
-			binary_path = "\"" + binary_path + "\"";
-			ffmpeg_binary_path = binary_path;
-			break;
-		}
+	// Windows always have ffmpeg
+	std::string binary_path = etl::dirname(synfig::get_binary_path(".")) + "/ffmpeg.exe";
+	if (g_access(binary_path.c_str(), F_OK ) != -1 ) { // File found
```

Comment:
Maybe we should use `X_OK` ? It checks for existence and execution permission.

Reply:
Windows file systems do not have an execution bit for the file (https://docs.microsoft.com/en-us/windows/win32/fileio/file-attribute-constants). So it looks like this is not necessary.

Reply:
Oh, Ok.
I just thought it because, at my work, the IT crew setup somehow some folders contents can't be executed :) 
But I don't know if this function would catch those cases.

Reply:
Very interesting. I will take a look.

Reply:
@rodolforg found details here: https://developer.gnome.org/glib/stable/glib-File-Utilities.html#g-access

> On Windows, the file protection mechanism is not at all POSIX-like, and the underlying function in the C library only checks the FAT-style READONLY attribute, and does not look at the ACL of a file at all. This function is this in practise almost useless on Windows. Software that needs to handle file permissions on Windows more exactly should use the Win32 API.